ios/objective-c/AmpliObjectiveCSampleApp/AmpliObjectiveCSampleApp/AppDelegate.m

```diff
@@ -0,0 +1,90 @@
+//
+//  AppDelegate.m
+//  AmpliObjectiveCSampleApp
+//
+//  Created by Qingzhuo Zhen on 11/24/21.
+//
+
+#import "AppDelegate.h"
+#import "Ampli.h"
+
+@interface AppDelegate ()
+
+@end
+
+@implementation AppDelegate
+
+
+- (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions {
+    //    '[Ampli instance]' is the default instance of Ampli
+    //
+    //    When you pull your tracking plan you can use the defaults and call load() without arguments
+    //    This requires connecting your account via `ampli pull` which will set you API key in the generated Ampli SDK
+    //
+    //    OR Specify a AmpliEnvironment
+    //    [ampli load:[LoadOptions initWithEnvironment:development]];
+    //
+    //    OR Provide a specific Amplitude API key
+    //    [ampli load:[LoadOptions initWithApiKey:@"Custom api key"]];
+    //
+    //    OR Use an existing Amplitude instance
+    //    requires `#import "Amplitude.h"`
+    //    Amplitude *instance = [Amplitude instanceWithName:@"instanceName"];
+    //    [instance initializeApiKey:@"Custom api key"];
+    //    [ampli load:[LoadOptions initWithAmplitudeInstance:instance]];
+    //
+    //    For testing you can disable ampli
+    //    [ampli load:[LoadOptions initWithDisabled:ENV.IS_TESTING ? YES : NO]];
+    //
+    //    Make as many Ampli instances as you want
+    //    Ampli ampli2 = [Ampli new];
+    //    [ampli2 load:[LoadOptions initWithApiKey:@"Custom api key"]];
+    NSString *apiKey = [[[NSProcessInfo processInfo] environment] objectForKey:@"AMPLITUDE_API_KEY"];
+    Ampli *ampli = [Ampli instance];
+    [ampli load:[LoadOptions builderBlock:^(LoadOptionsBuilder *builder) {
+        builder.apiKey =apiKey;
+    }]];
+    IdentifyProperties *identifyProperties = [IdentifyProperties new];
+    identifyProperties.optionalArray = [NSArray arrayWithObjects:@"optional string", nil];
+    identifyProperties.requiredNumber = @22;
+    [ampli identify:@"ampli-objc-user" properties:identifyProperties];
+    [ampli eventNoProperties];
+    NSMutableDictionary *extraDict = [NSMutableDictionary new];
+    [extraDict setObject:@"extra test"
+              forKey:@"test"];
+    EventMaxIntForTestProperties *eventMaxIntForTestProperties = [EventMaxIntForTestProperties new];
+    eventMaxIntForTestProperties.intMax10 = @20;
+    [ampli eventMaxIntForTest:eventMaxIntForTestProperties extra:extraDict];
```

Comment:
This section is pretty dense. Can you please add some space between each Event to make it a little easier to read.

Reply:
Sure, will do.